spec/string_foundation/length_spec.rb

```diff
@@ -0,0 +1,36 @@
+# ==============================================================================
+# SPEC - STRING FOUNDATION - LENGTH
+# ==============================================================================
+# frozen_string_literal: true
+describe "[ Length Methods ]" do
+
+  describe "COMPARE CHARACTERS LENGTH" do
+    context "when the argument is String class," do
+      it { expect{ "string".length?("6") }.to raise_error ArgumentError }
+    end
+
+    context "when characters length and argument are the same," do
+      it { expect("string".length?(6)).to be true }
+    end
+
+    context "when characters length and argument are not the same," do
+      it { expect("string".length?(5)).to be false }
+    end
+
+    context "when character length are within range of argument," do
+      it {
```

Comment:
Use `do ... end` for multiline block.

Reply:
Also, specify description string for `it` method when using multiline block. In general, starting with a word `"should"` .

```rb
# OK
# (Right English: It should be true.)
it "should be true" do
  expect(xxx).to be true
end
```